enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,231 @@
+
+2. Published CoreOS Images to the IBM Cloud Pre-Authorized Default Images
+This approach would involve outputting our CoreOS OVA images in a queue where they can be pre-loaded into the IBM Cloud
+so that they are available as soon as the Virtual Server Instance is available. This is significantly quicker than the
+method above but has several drawbacks. For one, these default images need to be pre-authorized in order for them to be
+added to the IBM Cloud. This is currently a manual process. Additionally, while we could build automation to solve this,
+it's unclear how would allow the user to point to the correct CoreOS image in the default list, since they would have to
+know the image name and details before their instance is created. Finally, we don't want to muddy up the default image
+list with an ever growing list of CoreOS options. This pruning would need to be automated some other mechanism would
+need to be in place to protect the user experience.
+
+#### DHCP
+The Power VS environment doesn't currently have a way of handling DHCP within the environment. We've asked their team to
+look into how they can provide this, since static IPs are currently not supported for IPI installs.
+
+#### Load Balancing
+When it came to load balancer, the best option we came up with was to use the Direct Link IBM Cloud offering, since it
+allows us to provide an application load balancer to the cluster that can talk directly to the private network. This
+would come at no additional cost to the user, since links are already provided with subscriptions to Power VS.
+
+### Risks and Mitigations
+
+The Multi-Arch team has already mitigated much of the risks involved with coordinating a new ppc64le platform. We have
+existing engagements with many of the teams that will would be working with to add this platform. For instance, we
+already have a hardware environment capable of buiding the OVA images, a dedicated CI team to build out and analyze the
+development focused testing, and the underlying content release content we'd need build for ppc64le by the ART team.
+
+## Design Details
+
+The Power Systems Virtual Servers (Power VS) are separated from the rest of the IBM Cloud servers with separate networks
+and direct-attached storage. The internal networks are fenced but offer connectivity options to IBM Cloud infrastructure
+or on-premises environments.
+Both public and private network is supported for instances (VMs). For public networks only limited ports are opened for
+inbound access and this is not configurable.
+
+Basic Walkthrough:
+1. Create a paid IBM Cloud Account (PowerVS - or any infrastructure service, is not available for free accounts).
+2. Create Power VS Service instance in specific region
+3. Create private network in the service instance
+4. Upload/Create the required boot images.
+5. Create VM by specifying #CPU, #Memory, Machine type, Boot Image, additional disk volume, network (public, private)
```

Comment:
In addition to flavors, there are other aspects of provisioning compute:

Dedicated vs Shared
Class of server (s922 vs e980)

And provisioning storage
Tier1 or Tier3

Should these be baked into the installer or do these controls get exposed?

Reply:
@manojnkumar is there a high level install type that can achieve this? for example aws has instance types (like m4, m5) which is essentially maps to a list of resources in a particular tier. the installer can default to a type and it can also expose an option to override this if necessary.

Reply:
@manojnkumar since these are currently asked when creating a Power VS VM Instance, and also since they affect cost,  My thinking was to expose memory, cores, system type, etc., in the platform details of the install config. 
@Prashanth684 there's not anything like that to my knowledge.

Reply:
Added a section in open questions to discuss this topic further.

Reply:
@Prashanth684 no currently it's not there.